### PR TITLE
Roadmap/Q1 2021

### PR DESCRIPTION
**Pull request by breathingdust**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

File: docs/roadmaps/2020_November_to_January.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/hashicorp/terraform-provider-aws/milestone/138).
+
+This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
+
+From [August through October](docs/roadmaps/2020_August_to_October.md), we committed to adding support for EventBridge, ImageBuilder , LakeFormation and Serverless Application Repository as new service offerings. We were able to deliver EventBridge within that time frame. Unfortunately for a number of reasons we weren’t able to release ImageBuilder, LakeFormation and Serverless Application Repository. That said, they are in progress and on track for release in early November.
```
Comment (bflad) [inline docs/roadmaps/2020_November_to_January.md:11]: Link fix:

```suggestion
From [August through October](2020_August_to_October.md), we committed to adding support for EventBridge, ImageBuilder , LakeFormation and Serverless Application Repository as new service offerings. We were able to deliver EventBridge within that time frame. Unfortunately for a number of reasons we weren’t able to release ImageBuilder, LakeFormation and Serverless Application Repository. That said, they are in progress and on track for release in early November.
```

File: docs/roadmaps/2020_November_to_January.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Roadmap:  November 2020 - January 2021
+
+Every few months, the team will highlight areas of focus for our work and upcoming research.
+
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [Core Services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors, are not available we will create the resources and implementation ourselves.
```
Comment (bflad) [inline docs/roadmaps/2020_November_to_January.md:5]: Link fix:

```suggestion
We select items for inclusion in the roadmap from the Top 10 Community Issues, [Core Services](../CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors, are not available we will create the resources and implementation ourselves.
```

**Review by maryelizbeth — APPROVED** [review]

**Review by maryelizbeth — APPROVED** [review]

File: ROADMAP.md [hunk 3]
```diff
@@ -8,31 +8,64 @@ Each weekly release will include necessary tasks that lead to the completion of
...
+_[CloudWatch Synthetics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html) You can use Amazon CloudWatch Synthetics to create canaries, configurable scripts that run on a schedule, to monitor your endpoints and APIs. Canaries follow the same routes and perform the same actions as a customer, which makes it possible for you to continually verify your customer experience even when you don't have any customer traffic on your applications. By using canaries, you can discover issues before your customers do._
 
 Support for AWS SSO Permission Sets will include:
```
Comment (shuheiktgw) [inline ROADMAP.md:45]: I think this should be `CloudWatch Synthetics`?

Comment (maryelizbeth) [reply ROADMAP.md:45]: Thanks! Good Catch - i'll fix it!

Comment (maryelizbeth) [inline ROADMAP.md:45]:
```suggestion
Support for CloudWatch Synthetics will include:
```

**Review by maryelizbeth — CHANGES_REQUESTED** [review]

File: docs/roadmaps/2020_November_to_January.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+### AWS SSO Permission Sets
+Issue: [#15108](https://github.com/hashicorp/terraform-provider-aws/issues/15108)
+
+_[AWS SSO](https://docs.aws.amazon.com/singlesignon/latest/APIReference/welcome.html) account assignment APIs enable you to build automation to create and update permissions that align with your company's common job functions. You can then assign the permissions to users and groups to entitle them for access in their required accounts. For example, you can give your developers broad control over resources in developer accounts, and limit that control to authorized operations personnel in production accounts. The new AWS CloudFormation support enables you to automate account assignments as you build new accounts. You can also use the APIs to decode user and group names from the unique identifiers that appear in AWS CloudTrail logs._
+
+Support for AWS SSO Permission Sets will include:
+
+New Resource(s):
+- aws_sso_permission_set
```
Comment (maryelizbeth) [inline docs/roadmaps/2020_November_to_January.md:25]:
```suggestion

- aws_sso_permission_set
```

**Review by maryelizbeth — CHANGES_REQUESTED** [review]

**Review by maryelizbeth — APPROVED** [review]

File: docs/roadmaps/2020_November_to_January.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+### Research Topics
+
+Research topics include features, architectural changes, and ideas that we are pursuing in the longer term that may significantly impact the core user experience of the AWS provider. Research topics are discovery only and are not guaranteed to be included in a future release.
+
+We are interested in your thoughts and feedback about the proposals below and encourage you to comment on the linked issues or schedule time with @maryelizbeth via the link on her GitHub profile to discuss.
+
+#### Default Tags Implementation Design
+Issue: [#7926](https://github.com/hashicorp/terraform-provider-aws/issues/7926)
+
+After completing user research and an internal review of our research conclusions, we will begin conducting engineering research and publish an RFC to address the implementation of this feature. Once the RFC has been approved, we will update the community with our plans for Default Tags.
```
Comment (maryelizbeth) [inline docs/roadmaps/2020_November_to_January.md:88]:
```suggestion
After completing user research and an internal review of our research conclusions, we will begin conducting engineering research and publish an RFC to address the implementation of this feature. Once the RFC has been approved, we will update the community with our plans for Default Tags.
```

File: docs/roadmaps/2020_November_to_January.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+The Elasticache work will begin with a research spike to ensure that the we can solve the following issues without introducing breaking changes into the provider:  
+
+- [#14959](https://github.com/hashicorp/terraform-provider-aws/issues/14959): Research Spike: Elasticache Service Fixes and Improvements
+- [#12708](https://github.com/hashicorp/terraform-provider-aws/issues/12708): resource/aws_elasticache_replication_group: Add MultiAZ support
+- ~[#13517](https://github.com/hashicorp/terraform-provider-aws/issues/13517): Feature Request: `aws_elasticache_cluster` allow auto-minor-version-upgrade to be set~ This parameter is not enabled in the AWS API.
+- [#5118](https://github.com/hashicorp/terraform-provider-aws/issues/5118): support setting primary/replica AZ attributes inside NodeGroupConfiguration for RedisClusterModelEnabled
+
+### Workflow Improvements
+
+We’ll also be tackling some of the top reported issues in the provider that are causing disruptions to high priority workflows:
```
Comment (maryelizbeth) [inline docs/roadmaps/2020_November_to_January.md:57]:
```suggestion
We’ll also be tackling some of the top reported issues in the provider that are causing disruptions to high priority workflows:
```

**Review by maryelizbeth — CHANGES_REQUESTED** [review]

**Review by maryelizbeth — APPROVED** [review]

**Pull request merged**